apps/financial-aid/web-veita/src/components/FilesListWithHeader/FilesListWithHeader.tsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react'
 import { Text, Box } from '@island.is/island-ui/core'
 import cn from 'classnames'
-import { ApplicationFile } from '@island.is/financial-aid/shared'
-import { FileList } from '@island.is/financial-aid-web/shared/src/components'
+import { ApplicationFile } from '@island.is/financial-aid/shared/index'
+import FileList from '@island.is/financial-aid/shared/components/FileList/FileList'
```

Comment:
Strange to get some stuff from `shared/index` and some stuff from `shared/deep/path`.

We should generally encapsulate libs and avoid deep importing from them. Better to create more libs (eg `@island.is/financial-aid/components`) and import stuff from each lib's root index.ts module.

Then we also don't need this `/index` part.

Fyi, the next version of NX adds a lint error when using absolute imports instead of relative imports for modules in the same workspace. I've also been suggesting for the `web` project to move away from absolute deep imports from `web` to splitting it up into libs.